src/openlifu/bf/__init__.py

```diff
@@ -1,5 +1,15 @@
-from .delay_methods import DelayMethod
 from .apod_methods import ApodizationMethod
+from .delay_methods import DelayMethod
 from .focal_patterns import FocalPattern, SinglePoint, Wheel
 from .pulse import Pulse
 from .sequence import Sequence
+
+__all__ = [
```

Comment:
Thanks, I will edit the commit message. It's part of "remove unused imports" because adding things to `__all__` is a way to indicate the intention that something is being imported in order to be exported, which is needed for the linting to pass after removal of the unused-import exception